federation/cluster/template.go

```diff
@@ -0,0 +1,52 @@
+package main
```

Comment:
Thank you for doing this instead of writing a hairy bash thing! Can you add a doc commment here?